support-frontend/assets/components/subscriptionCheckouts/thankYou/marketingConsentContainerGift.ts

```diff
@@ -7,15 +7,14 @@ import { sendTrackingEventsOnClick } from 'helpers/productPrice/subscriptions';
 import type { CsrfState } from 'helpers/redux/checkout/csrf/state';
 import { getEmail } from 'helpers/subscriptionsForms/formFields';
 import type { CheckoutState } from 'helpers/subscriptionsForms/subscriptionCheckoutReducer';
-import type { Action } from 'helpers/user/userActions';
 
 const mapStateToProps = (state: CheckoutState) => ({
 	confirmOptIn: state.page.checkoutForm.marketingConsent.confirmOptIn,
 	email: getEmail(state),
 	csrf: state.page.checkoutForm.csrf,
 });
 
-function mapDispatchToProps(dispatch: Dispatch<Action>) {
+function mapDispatchToProps(dispatch: Dispatch) {
```

Comment:
These look like a simplification of dispatching actions would guess?

Reply:
No, it's just a simplification of the type because that particular `Action` type no longer exists and `mapDispatchToProps` doesn't play very well with RTK's store-specific `Dispatch` types.